main.js

```diff
@@ -392,9 +392,13 @@ var obstacles = [
   [obstacleSpike, 300]
 ];
 
-var drawBoundingBox = function(obstacle) {
+var drawBoundingBox = function(obstacle, xmin, ymin) {
```

Comment:
Would using `xmin` and `xmax` here also work for the saw obstacle? Let's say we want to only check collisions with the saw blade but not the arm it's sitting on. Then the bounding box wouldn't be sitting on the floor.